### ISSUE TITLE
video/voodoo.cpp: VGA passthrough support for Voodoo 1 & 2

### ISSUE THREAD

**Issue by angelosa** [mamedev/mame#11343]
Voodoo 1 and 2 fbiInit0 bit 0 drives to external pin `vga_pass`, required for switching between multiple devices over the same screen. The catch is that this should also swap between who controls the CRT, which sounds like  #5801 league.
fbinit3 bit 11 is a read only bit for VGA_PASS reset value. 

Known systems requiring this which can be used for :
- pf2012 bootscreen CPLD
- savquest S3 trio64uv

This doesn't apply to Voodoo Banshee and Voodoo 3, which has their own 3dfx VGA controller sub-device.

**Comment by Hydreigon223** [comment]
Does racingj in nwk-tr.cpp count as well? That game has a tendency to display both CG board tilemap customs during POST which explains why you can't see some text during POST (and in earlier versions when this was displayed as two screens, you could see the missing text on the other screen). Could be an entirely different can of worms if I'm wrong.

**Comment by angelosa** [comment]
Possibly ya, hangplt is another one (where it definitely wants the back layer to be displayed for the cross hatch test in service mode, in place of Voodoo layer being full white there).

**Comment by cuavas** [comment]
Switching who controls screen timings is something MAME really isn’t good at.

The only way I can think of implementing it right now would be to have some member function on the “cooperating” device (i.e. the VGA’s CRTC in this case) like `set_controls_crt(bool)` where passing `false` causes it to suspend setting screen timing parameters, and passing `true` pushes screen parameters immediately if it was previously false and resumes setting screen timing parameters.

**Comment by Hydreigon223** [comment]
I ask gticlub.cpp and nwk-tr.cpp to be removed here and saved for a different issue after analyzing various signals on the 3dfx ICs (at least on a hornet video pcb).

- VGA passthrough pin itself isn't used here; I looked everywhere.
- DAC data pins from the 3dfx IC go to the tilemap IC; this is also the same with earlier pre-3dfx video boards (from testing on a gticlub video pcb I own).
- DAC Vsync and Hsync pins aren't used unlike most cards and Midway Seattle; The video connector signals all (eventually) lead to the tilemap IC.

So in conclusion, not only was I incorrect but this was the exact opposite of what I expected with the voodoo "feeding" through the tilemap IC. This indeed is an entirely different can of worms.
